inject/src/main/java/io/micronaut/context/AbstractBeanContextConditional.java

```diff
@@ -38,8 +38,6 @@
 @Internal
 abstract class AbstractBeanContextConditional implements BeanContextConditional, AnnotationMetadataProvider {
 
-    static final Logger LOG = LoggerFactory.getLogger(Condition.class);
```

Comment:
why do we need to do this change?

Reply:
To avoid the logs being initialized at build time